packages/core/src/decorators/__tests__/medi-type-examples-override.test.ts

```diff
@@ -0,0 +1,432 @@
+import { makeConfig, parseYamlToDocument, yamlSerializer } from '../../../__tests__/utils';
+import { outdent } from 'outdent';
+import { bundleDocument } from '../../bundle';
+import { BaseResolver } from '../../resolve';
+
+describe('oas3 media-type-examples-override', () => {
+  expect.addSnapshotSerializer(yamlSerializer);
+  it('should override response example', async () => {
+    const testDocument = parseYamlToDocument(
+      outdent`
+            openapi: 3.0.0
+            paths:
+              /pet:
+                get:
+                  operationId: getUserById
+                  responses:
+                    200:
+                      description: json
+                      content:
+                        application/json:
+                          example:
+                            def:
+                              value:
+                                a: test
+                      
+    `
+    );
+    const { bundle: res } = await bundleDocument({
+      document: testDocument,
+      externalRefResolver: new BaseResolver(),
+      config: await makeConfig(
+        {},
+        {
+          'media-type-examples-override': {
+            operationIds: {
+              getUserById: {
+                responses: {
+                  '200': 'packages/core/src/decorators/__tests__/resources/request.yaml',
+                },
+              },
+            },
+          },
+        }
+      ),
+    });
+
+    expect(res.parsed).toMatchInlineSnapshot(`
+      openapi: 3.0.0
+      paths:
+        /pet:
+          get:
+            operationId: getUserById
+            responses:
+              '200':
+                description: json
+                content:
+                  application/json:
+                    example:
```

Comment:
This seems like it is a mistake. I think this means to be `examples`?

Reply:
Yes, thanks